picky/src/Components/Pages/EntryScreen.js

```diff
@@ -1,11 +1,11 @@
 import React, { Component } from 'react';
-import bag from './bag.png';
-import EntryScreen from './Components/EntryScreen';
-import './App.css';
+import bag from '../../bag.png';
+import Login from '../Basic/Login';
+import '../../App.css';
```

Comment:
Look into setting up a root directory to avoid having to use relative paths like this.

Check out [this blog post](http://moduscreate.com/es6-es2015-import-no-relative-path-webpack/).